package/libs/openssl/Makefile

```diff
@@ -156,6 +156,8 @@ else
     OPENSSL_TARGET:=linux-mips-openwrt
 #  else ifeq ($(CONFIG_arm)$(CONFIG_armeb),y)
 #    OPENSSL_TARGET:=linux-armv4-openwrt
+  else ifeq ($(BOARD),ipq806x)
+    OPENSSL_TARGET:=linux-armv4-openwrt
```

Comment:
I think this should not be based on the target, it should be based on the CPU type